cmd/cdi/cmd/validate.go

```diff
@@ -40,6 +41,13 @@ were reported by the registry.`,
 			return
 		}
 
+		fmt.Printf("CDI Registry has errors:\n")
+		for path, specErrors := range cdiErrors {
+			fmt.Printf("Spec file %s:\n", path)
+			for idx, err := range specErrors {
+				fmt.Printf("  %d: %v\n", idx, strings.TrimSpace(err.Error()))
```

Comment:
Thanks for the update. As a matter of interest, do we want to use tabs instead of spaces here so that the columns are aligned? This really isn't a blocker though, and I'm happy to approve as is.

Reply:
Instead of using tabs I now changed this to `%3d` so it will reserve 3 chars for the idx. Will get unaligned if there are more than 999 errors

Reply:
Actually, that would be 999 errors per spec. I changed it to `%2d` so it will get unaligned after 99 errors per spec